rsp_jupyter_extensions/handlers/execution.py

```diff
@@ -20,25 +20,32 @@ def rubinexecution(self) -> dict[str, str]:
 
     @tornado.web.authenticated
     def post(self) -> None:
-        """
-        POST the contents of a notebook and get back the rendered,
-        executed notebook.
+        """Handle ``POST /rubin/execution``.
+
+        This handler executes a notebook, and returns the rendered notebook,
+        along with any resources and errors that occurred during execution.
 
-        There are two supported formats.  The first is simply the text
-        of an ipynb file.  This is expected to be the common use case.
+        **Request body.**
+        The request body can take two forms:
+        1. A string containing the text of an ipynb file.
+        2. A JSON-encoded dict containing a ``notebook`` key (the ipynb as a
+           string) and a ``resources`` key (a JSON-encoded dict of resources).
+           The second form is used less often, but is useful for passing
+           resources to the notebook that are not part of the notebook itself.
 
-        The second is a JSON representation of a dict containting a
-        notebook and associated resources; the notebook contents (a string
-        containing an ipynb file) will be in the "notebook" key and
-        the resources will be a string in the key "resources" representing
-        a JSON-encoded dict).
+        **Request headers.**
+        Set the ``X-Kernel-Name`` header to the name of the kernel to use for
+        execution.
         """
         input_str = self.request.body.decode("utf-8")
+        kernel_name = self.request.headers.get("X-Kernel-Name", None)
```

Comment:
I think `None` is a better choice than `"LSST"` because this stuff goes in the base container, which may be used to construct images with non-DM-stack payloads, and those would not even have an `LSST` kernel.

But I had to think about it for a few minutes to make up my mind.

Reply:
Keeping None as the default maintains the current behaviour, so I think it makes sense.